tests/general/SpaceConsistencyBearTest.py

```diff
@@ -62,13 +62,11 @@ def test_enforce_newline_at_eof(self):
                              "    print('funny')\n",
                              "    print('funny end.')\n"],
                             force_linebreaks=False)
-        self.check_validity(self.uut,
+        self.check_invalidity(self.uut,
                             [' no hello world'],
```

Comment:
@prachi1210 The same issue here too. Simply apply this patch to fix it directly.